tests/images/customize.sh

```diff
@@ -145,45 +145,118 @@ qemu_disk() {
 }
 
 assert() {
-    grep -Fq "$1" log
+    if grep -Fq "$1" log; then
+        echo "Assertion passed: '$1' found in log"
+        return 0
+    else
+        echo "Assertion failed: '$1' not found in log"
+        return 1
+    fi
+}
+
+assert() {
+   grep -Fq "$1" log
+}
+
+assert_not() {
+    if grep -Fq "$1" log; then
+        return 1
+    else
+        return 0
+    fi
 }
 
 check_live_noinstall() {
-    assert @applied-live-ign@
-    assert @applied-live-2-ign@
-    ! assert @applied-dest-ign@
-    ! assert @applied-dest-2-ign@
-    assert @did-not-install@
-    ! assert @preinst-1@
-    ! assert @preinst-2@
-    ! assert @postinst-1@
-    ! assert @postinst-2@
-    assert 'Adding "coreos-installer test certificate" to list of CAs'
+    echo "Checking assertion: @applied-live-ign@"
+    assert @applied-live-ign@ || { echo "Failed assertion: @applied-live-ign@"; return 1; }
```

Comment:
Not sure how much value these echo statements add on top of what the `assert()` function now prints.

Reply:
Ugh I changed my mind let me reverse uno this.

Reply:
Hmm, I think the other way would be cleaner, no? (I.e. keep the `echo`s in the `assert()` and no printing here). All the echos here are very mechanical.

Reply:
Yeah idk what I was thinking I reverse uno'd it.